src/Backoffice/Cities/App/Transformers/CityTransformer.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Cities\App\Transformers;
+
+use Flugg\Responder\Transformers\Transformer;
+use Lightit\Backoffice\Cities\Domain\Models\City;
+
+class CityTransformer extends Transformer
+{
+    /**
+     * @return array{id: int, name: string}
+     */
+    public function transform(City $city): array
+    {
+        return [
+            'id' => $city->id,
+            'name' => $city->name,
+        ];
+    }
+}
```

Comment:
You should return the arriving a departures flights.
This is how we do it:

```suggestion
class CityTransformer extends Transformer
{
    protected $load = [
        'arrivalFlights',
        'departureFlights'
    ];
    
    /**
     * @return array{id: int, name: string}
     */
    public function transform(City $city): array
    {
        return [
            'id' => $city->id,
            'name' => $city->name,
        ];
    }
}
```